### ISSUE TITLE
(BUG) Weight added for every item in inventory. Also suggestion?

### ISSUE THREAD

**Issue by TheComicRelief** [FelixAllard/SCP966LC#3]
Hi it's me again. The infinite weight bug is fixed and I am very glad. Anyway, it seems that having each item adds a great amount of weight to the player. I was 52 lbs with nothing, but having a shovel shoots it up to 171 lbs. By the time i have 6 (using hotbarplus), it's 700+ lbs. I wonder why killing it doesn't decrease the weight, it would be fun (and way less annoying) if it does.

**Comment by FelixAllard** [comment]
Oh so you mean, you'd want for the weight to be less explosive right?
Also, you mean that when you kill him, you don't lose your weight is that what I understand?

**Comment by Skyblade799** [comment]
Yeah, just encountered the enemy solo for the first time, and my weight skyrocketed to 350 lbs from an Airhorn and engine (no weight modifications/mod upgrades to change the base weight either, but I did have an extra flashlight slot and 2 slot extended hotbar), meaning I gained around 200+ lbs. Doesn't match the 52 lbs listed on the enemy page. Going to disable it until fixed.

**Comment by FelixAllard** [comment]
Yup! That is definitly a bug....
I will fix it soon, thank you for the report!
